decidim-core/spec/shared/with_promoted_participatory_processes_examples.rb

```diff
@@ -53,6 +53,7 @@
 
       expect(controller.helpers.promoted_participatory_processes.count).to eq(3)
       expect(controller.helpers.promoted_participatory_processes).not_to include(unpublished)
+      expect(controller.helpers.promoted_participatory_processes).not_to include(unpromoted)
```

Comment:
Updated via 143bed2. The best use case of `match_array` I've ever seen :laughing:.